apps/src/gamelab/TextConsole.jsx

```diff
@@ -0,0 +1,169 @@
+/** @file Text display over sprite lab visualization column */
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Transition} from 'react-transition-group';
+
+const lineHeight = 18;
+const maxHeight = lineHeight * 6;
+
+export const styles = {
+  hide: {
+    display: 'none'
+  },
+  console: {
+    display: '',
+    background: 'rgba(200,200,200,0.5)',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: 2,
+    width: '100%',
+    lineHeight: lineHeight,
+    transitionProperty: 'max-height',
+    transitionDuration: '1000ms'
+  },
+  expandButton: {
+    position: 'absolute',
+    right: 0,
+    zIndex: 3,
+    minWidth: lineHeight,
+    margin: 0,
+    border: 0,
+    padding: 0,
+    fontSize: 'inherit',
+    lineHeight: 'inherit',
+    borderRadius: 0,
+    fontFamily: 'monospace'
+  },
+  paragraphStyle: {
```

Comment:
nit: it would be a lot more descriptive to call this something like `line`

Reply:
changed to `text`